### ISSUE TITLE
SELinux Boolean domain_can_mmap_files does not exist on CentOS/RHEL prior to v7.6

### ISSUE THREAD

**Issue by techwizrd** [robertdebock/ansible-role-anaconda#4]
**Describe the bug**
The SELinux Boolean `domain_can_mmap_files` was [introduced in Red Hat Enterprise Linux 7.6](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/7.6_release_notes/index) and CentOS 7.6. The task `modify selinux settings` will fail on versions of RHEL/CentOS prior to 7.6.

**To Reproduce**
1. Run this role against a host running RHEL/CentOS 7.5 or earlier.
2. See error:
```
TASK [robertdebock.anaconda : modify selinux settings] ***************************************************************************************************************************************************************************************
fatal: [target_machine]: FAILED! => {"changed": false, "msg": "SELinux boolean domain_can_mmap_files does not exist."}
```

**Expected behavior**
The `modify selinux settings` task should be skipped on RHEL/CentOS 7.5 and below.

**Error**
Relevant Ansible snippet:
```
- name: modify selinux settings
  seboolean:
    name: domain_can_mmap_files
    state: yes
    persistent: yes
  when:
    - ansible_selinux.status is defined
    - ansible_selinux.status == "enabled"
```
and the resulting error:
```
TASK [robertdebock.anaconda : modify selinux settings] ***************************************************************************************************************************************************************************************
fatal: [target_machine]: FAILED! => {"changed": false, "msg": "SELinux boolean domain_can_mmap_files does not exist."}
```

**Environment**
- Control node OS: macOS 10.14.1
- Control node Ansible version: 2.8.0
- Managed node OS: CentOS 7.4

**Additional context**
This may occur on other operating systems where SELinux does not have the SELinux Boolean `domain_can_mmap_files`.

**Comment by robertdebock** [comment]
Okay, I'll test this and write a fix.

Thanks for you feedback!

**Comment by techwizrd** [comment]
I wrote and tested a fix, and opened PR #5 with the fixed code.
